apps/system/utils/kdbg_date.c

```diff
@@ -127,6 +127,9 @@ static inline int date_settime(int argc, char **args)
 	FAR char *saveptr;
 	long result;
 	int ret = OK;
+#ifdef CONFIG_LIBC_LOCALTIME
+	tm.tm_isdst = 0;
```

Comment:
initialized the local copy all the members associated with CONFIG_LIBC_LOCALTIME) || CONFIG_TIME_EXTENDED and then set coresponding values